examples/wave-eager-mpi.py

```diff
@@ -112,15 +116,21 @@ def main(snapshot_pattern="wave-eager-{step:04d}-{rank:04d}.pkl", restart_step=N
 
     discr = EagerDGDiscretization(actx, local_mesh, order=order,
                                   mpi_communicator=comm)
-
-    if local_mesh.dim == 2:
-        # no deep meaning here, just a fudge factor
-        dt = 0.7 / (nel_1d*order**2)
-    elif dim == 3:
-        # no deep meaning here, just a fudge factor
-        dt = 0.4 / (nel_1d*order**2)
+    if not constant_cfl:
+        if local_mesh.dim == 2:
+            # no deep meaning here, just a fudge factor
+            dt = 0.7 / (nel_1d*order**2)
+        elif dim == 3:
+            # no deep meaning here, just a fudge factor
+            dt = 0.4 / (nel_1d*order**2)
+        else:
+            raise ValueError("don't have a stable time step guesstimate")
```

Comment:
I was actually going to say we should replace this hokey junk with just constant DT mode, but as I am writing now I think I agree with your notion to leave it alone.  It sort of auto-tunes the constant DT (hokey or not), obviating the user's configuration-specific choice for DT.

Let's see what shakes out if we ask the author.

Reply:
Ok, it looks like we want to go ahead and rip out this stuff.   Can you please replace the kludge here with a single call to the timestep estimator?   Actually - I've done a dumb thing to you.  The wavespeed and grid never change in this simulation (unlike a fluid sim) - so a single call to the timestep estimator is sufficient for the wave examples.  They do not need a constant CFL mode.  Sigh :(

Reply:
I just removed it. I also realized that dt was constant after I pushed up- I'm not exactly sure why I kept the old dt code in there in the first place.